src/cache/cache_las.c

```diff
@@ -920,9 +920,10 @@ __las_sweep_init(WT_SESSION_IMPL *session, WT_CURSOR *las_cursor)
      * If no files have been dropped and the lookaside file is empty, there's nothing to do.
      */
     if (cache->las_dropped_next == 0) {
-        if (__wt_las_empty(session))
+        if (__wt_las_empty(session)) {
             ret = WT_NOTFOUND;
-        goto err;
+            goto err;
+        }
```

Comment:
This change really should be in a different ticket that's merged into develop. We want to get this change backported to MongoDB 4.2 and earlier

Reply:
ok, I will fix the above change in the develop ticket and merge it here.

Reply:
Done.